<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VFR_algorithmClass</class>
 <widget class="QWidget" name="VFR_algorithmClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>验证Demo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,5,1,4">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>04b_21</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>监控人脸识别算法验证平台</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_video">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>375</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTextBrowser" name="textBrowser_log"/>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_selectVideo">
        <property name="text">
         <string>选择视频</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_openCamera">
        <property name="text">
         <string>开启摄像头</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cleanLog">
        <property name="text">
         <string>清空日志</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox_saveSnap">
        <property name="text">
         <string>保存抓拍</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_querySnap">
        <property name="text">
         <string>查看抓拍</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget_snap"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VFR_algorithm.qrc"/>
 </resources>
 <connections/>
</ui>
